src/app/thankYou/thankYou.component.js

```diff
@@ -43,6 +49,14 @@ class ThankYouController{
         if(this.purchase.paymentMeans.self.type === 'elasticpath.purchases.purchase.paymentmeans'){ //only credit card type has billing address
           this.billingAddress = this.orderService.formatAddressForTemplate(this.purchase.paymentMeans['billing-address'].address);
         }
+
+        // Display Account Benefits Modal when GUEST checkout
```

Comment:
Does this show the modal immediately on checkout? I thought that's what the button in the accountBenefits panel was for... It just seems like there are multiple ways to show the accountBenefits with duplicate code.